tests/sqlparser_mysql.rs

```diff
@@ -216,6 +216,260 @@ fn parse_create_table_with_minimum_display_width() {
+                        value: Expr::Function(Function {
+                            name: ObjectName(vec![Ident::new("VALUES".to_string()),]),
+                            args: vec![FunctionArg::Unnamed(Expr::Identifier(Ident::new(
+                                "description"
+                            )))],
+                            over: None,
+                            distinct: false
+                        })
+                    },
+                    Assignment {
+                        id: vec![Ident::new("perm_create".to_string())],
+                        value: Expr::Function(Function {
+                            name: ObjectName(vec![Ident::new("VALUES".to_string()),]),
+                            args: vec![FunctionArg::Unnamed(Expr::Identifier(Ident::new(
+                                "perm_create"
+                            )))],
+                            over: None,
+                            distinct: false
+                        })
+                    },
+                    Assignment {
+                        id: vec![Ident::new("perm_read".to_string())],
+                        value: Expr::Function(Function {
+                            name: ObjectName(vec![Ident::new("VALUES".to_string()),]),
+                            args: vec![FunctionArg::Unnamed(Expr::Identifier(Ident::new(
+                                "perm_read"
+                            )))],
+                            over: None,
+                            distinct: false
+                        })
+                    },
+                    Assignment {
+                        id: vec![Ident::new("perm_update".to_string())],
+                        value: Expr::Function(Function {
+                            name: ObjectName(vec![Ident::new("VALUES".to_string()),]),
+                            args: vec![FunctionArg::Unnamed(Expr::Identifier(Ident::new(
+                                "perm_update"
+                            )))],
+                            over: None,
+                            distinct: false
+                        })
+                    },
+                    Assignment {
+                        id: vec![Ident::new("perm_delete".to_string())],
+                        value: Expr::Function(Function {
+                            name: ObjectName(vec![Ident::new("VALUES".to_string()),]),
+                            args: vec![FunctionArg::Unnamed(Expr::Identifier(Ident::new(
+                                "perm_delete"
+                            )))],
+                            over: None,
+                            distinct: false
+                        })
+                    },
+                ])),
+                on
+            );
+        }
+        _ => unreachable!(),
+    }
+}
+
+#[test]
+fn parse_update_with_joins() {
+    let sql = "UPDATE orders AS o JOIN customers AS c ON o.customer_id = c.id SET o.completed = true WHERE c.firstname = 'Peter'";
```

Comment:
I didn't find this type of syntax in the MySql manual https://dev.mysql.com/doc/refman/8.0/en/update.html

However, I tried such a thing in `mysql` and it worked 🤯 

```sql
mysql> create table items(id int, price double);
Query OK, 0 rows affected (0.01 sec)

mysql> create table month(id int, price double);
Query OK, 0 rows affected (0.00 sec)

mysql> UPDATE items,month SET items.price=month.price
    -> WHERE items.id=month.id;
Query OK, 0 rows affected (0.01 sec)
Rows matched: 0  Changed: 0  Warnings: 0

mysql> UPDATE items JOIN month ON items.id = month.id SET items.price=month.price WHERE items.id=month.id;
Query OK, 0 rows affected (0.00 sec)
Rows matched: 0  Changed: 0  Warnings: 0
```